Source/Core/DolphinQt2/Translate.h

```diff
@@ -0,0 +1,261 @@
+// Copyright 2015 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <QString>
+
+
+class Translate final
+{
+	public:
+	static void setLanguage(int languageID);
+	static void init();
+	#define _(string) Translate::translate(string)
```

Comment:
This really shouldn't be provided as a define, and we should have the full function name specified (this is a pro of Qt's system, as it doesn't require having a define pollute every namespace).